documentation/docs/libraries/rust/api_reference.md

```diff
@@ -6,10 +6,11 @@ keywords:
 - rust
 - cargo
 - crate
+- reference
 ---
 # API Reference
 
-You can visit the [Rust documentation](https://wallet-lib.docs.iota.org/docs/specification) to learn more about the API Reference.
+You can visit our [Rust documentation](https://wallet-lib.docs.iota.org/docs/specification) to learn more about the API Reference.
```

Comment:
this link doesn't work, it may work but we'd have to get the following pr approved and published first https://github.com/iotaledger/wallet.rs/pull/931